app/components/markdown/markdown_table/markdown_table.js

```diff
@@ -0,0 +1,146 @@
+
+const MAX_HEIGHT = 120;
+
+export default class MarkdownTable extends React.PureComponent {
+    static propTypes = {
+        children: PropTypes.node.isRequired,
+        navigator: PropTypes.object.isRequired,
+        theme: PropTypes.object.isRequired
+    };
+
+    static contextTypes = {
+        intl: intlShape.isRequired
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            contentCutOff: true
+        };
+    }
+
+    handlePress = wrapWithPreventDoubleTap(() => {
+        const {navigator, theme} = this.props;
+
+        navigator.push({
+            screen: 'Table',
+            title: this.context.intl.formatMessage({
+                id: 'mobile.routes.table',
+                defaultMessage: 'Table'
+            }),
+            animated: true,
+            backButtonTitle: '',
+            passProps: {
+                renderRows: this.renderRows
+            },
+            navigatorStyle: {
+                navBarTextColor: theme.sidebarHeaderTextColor,
+                navBarBackgroundColor: theme.sidebarHeaderBg,
+                navBarButtonColor: theme.sidebarHeaderTextColor,
+                screenBackgroundColor: theme.centerChannelBg
+            }
+        });
+    });
+
+    handleContentHeightChange = (contentWidth, contentHeight) => {
+        this.setState({
+            contentCutOff: contentHeight >= MAX_HEIGHT
+        });
+    }
+
+    renderRows = (drawBottomBorder = true) => {
+        const style = getStyleSheet(this.props.theme);
+
+        const tableStyle = [style.table];
+        if (drawBottomBorder) {
+            tableStyle.push(style.tableBottomBorder);
+        }
+
+        // Add an extra prop to the last row of the table so that it knows not to render a bottom border
+        // since the container should be rendering that
+        const rows = React.Children.toArray(this.props.children);
+        rows[rows.length - 1] = React.cloneElement(rows[rows.length - 1], {
+            isLastRow: true
+        });
+
+        return (
+            <View style={tableStyle}>
+                {rows}
+            </View>
+        );
+    }
+
+    render() {
+        const style = getStyleSheet(this.props.theme);
+
+        let moreIndicator = null;
+        if (this.state.contentCutOff) {
+            moreIndicator = (
+                <LinearGradient
+                    colors={[
+                        changeOpacity(this.props.theme.centerChannelColor, 0.0),
+                        changeOpacity(this.props.theme.centerChannelColor, 0.1)
+                    ]}
+                    style={style.moreIndicator}
+                />
+            );
+        }
+
+        return (
+            <TouchableOpacity onPress={this.handlePress}>
+                <ScrollView
+                    onContentSizeChange={this.handleContentHeightChange}
+                    style={style.container}
+                    scrollEnabled={false}
```

Comment:
Why wrap it in a `ScrollView` if it won't scroll no matter what? is a `View` not enough?

Reply:
I was having difficulty getting it to cut off at a certain max height with a View because it was trying to compress the content into the space instead of just cutting it off. Using a ScrollView that can't scroll gave me what I was looking for